src/typescript/azure-openapi-validator/rules/PathParameterMustNotBeEmpty.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MergeStates, OpenApiTypes, rules } from '../rule';
+export const PathParametersMustNotBeEmpty: string = "PathParametersMustNotBeEmpty";
+
+rules.push({
+  id: "R2067",
+  name: PathParametersMustNotBeEmpty,
+  severity: "error",
+  category: "RPCViolation",
+  mergeState: MergeStates.individual,
+  openapiType: OpenApiTypes.arm,
```

Comment:
You may want to check if it makes sense to extend it to Dataplane

Reply:
Initial  concern was for Management, but yes I don't see why wouldn't be for Dataplane. Will update